src/test/scala/memeid/V5Spec.scala

```diff
@@ -0,0 +1,23 @@
+package memeid
+
+import cats.effect._
+import cats.syntax.eq._
+
+import org.specs2.ScalaCheck
+import org.specs2.matcher.IOMatchers
+import org.specs2.mutable.Specification
+
+class V5Spec extends Specification with ScalaCheck with IOMatchers {
+  "V5 constructor" should {
+    "create same UUID for the same namespace/name" in {
```

Comment:
Maybe add a test that checks the opposite (distinct UUIDs for distinct namespace/name)

Reply:
Added, your comment helped catch a bug by the way :clap: